src/pages/index.js

```diff
@@ -0,0 +1,237 @@
+//good to go 4.30, submission attempt Sprint 9 #1 6.5.2024
+
+import Api from "../components/Api.js";
+import "../pages/index.css";
+import Card from "../components/Card.js";
+import FormValidator from "../components/FormValidator.js";
+import UserInfo from "../components/UserInfo.js";
+import PopUp from "../components/PopUp.js";
+import PopUpWithForm from "../components/PopUpWithForm.js";
+import Section from "../components/Section.js";
+import {
+  initialCards,
+  config,
+  profileEditButton,
+  profileEditModal,
+  profileTitle,
+  profileDescription,
+  profileTitleInput,
+  profileDescriptionInput,
+  profileEditForm,
+  //cardTemplate,
+  //cardListEl,
+  addCardButton,
+  addCardForm,
+  addCardModal,
+  addCardSubmit,
+  previewImage,
+  previewDescription,
+  //cardSelector,
+  profileAvatarButton,
+  profileAvatarModal,
+  profileAvatarForm,
+  cardDeleteButton,
+  cardDeleteModal,
+  cardDeleteForm,
+} from "../utils/constant.js";
+import PopUpWithImage from "../components/PopUpWithImage.js";
+import PopUpWithConfirmation from "../components/PopUpWithConfirmation.js";
+
+/* ------------------------------------------------------------------------------ */
+/* API */
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "bba6e655-3c43-4bc9-84b1-5706718b60cd",
+    "Content-Type": "application/json",
+  },
+});
+
+let section;
+
+api;
+
+Promise.all([api.getInitialCards(), api.getUserInfo()]).then(
+  ([cards, data]) => {
+    section = new Section(
+      {
+        items: cards,
+        renderer: (data) => {
+          const cardEl = renderCard(data);
+          section.addItem(cardEl);
+        },
+      },
+      ".gallery__cards"
+    );
+    section.renderItems();
+
+    userInfo.setUserInfo({
+      title: data.name,
+      description: data.about,
+    });
+
+    userInfo.setAvatar({ avatar: data.avatar });
+  }
+);
```

Comment:
Needs correction: please add the `catch` block here for handling the server errors